problem1.swift

```diff
@@ -2,4 +2,19 @@
 
 let input1 = Int(readLine()!)!
 let input2 = Int(readLine()!)!
-
+var i = 0
```

Comment:
For smaller programs, where you are using `i` very little _and_ only for iterating over arrays / looping n times, it's alright to use the variable name `i`. However, this can get very confusing in larger programs, so please try to give your variable and constant names descriptive names!